docs/howto/how_to_leverage_scenario_editor.md

```diff
@@ -0,0 +1,130 @@
+
+2. Once a map has been selected, you will have access to the editor pane on the right along with other tools as seen below:
+
+    ![](images/se_tools.png)
+
+
+You can navigate through the map using your arrow keys. Alternatively, you can right-click the mouse and drag it to move the map. If you are using a trackpad, you will have to two-finger click drag with two fingers.
+Lets understand each tool along with its purpose
+
+### General Action Tools
+
+The 4 General action tools can be found on the bottom right corner of the map. 
+
+1. **Zoom tool**: while you can use your trackpad to zoom in and out of the map, there exists the Zoom tool to help you zoom in and out of the map in case you do not have a trackpad ready.
+
+    ![](images/se_zoom1.png)
+
+2. **Re-center tool**: this tool allows you to locate you ego-car on the map even if you have moved away
+
+    ![](images/se_center1.png)
+
+3. **Ruler tool**: this tool allows you to measure the distance between two points. 
+
+    ![](images/se_ruler.png)  ![](images/se_distance.png)
+
+4. **Add Route tool**: you can set the ego car’s end point by clicking on the Add Route tool. Please note that the first version of the Scenario Editor does support the creation of multiple route points for the ego car.
+
+    ![](images/se_addroute.png)
+ 
+### Configuration Tools
+
+There are 3 types of configuration tools that you need to setup in order to create a scenario, which are listed on the left-side of the map - General, Ego-car and Obstacles
+
+![](images/se_config.png)
+
+#### General Configuration
+
+This configuration tool is selected by default upon the selection of a map. The form on your right requests general scenario informagtion like the scenario name, duration, road structure, ego-car behavior along with which metrics you would like to track. Please note some selections are required while some are based on your discretion. Once you have set and confirmed all the parameters, please proceed to the `Ego Car` configuration tool.
+
+You can learn more about what each parameter is by hovering over the `?` sign next to each parameter.
+
+#### Ego Car Configuration
+
+This configuration tool allows you to set your Ego car on the map and configure its parameters. As soon as you select the tool icon, you can then hover over the map and place the car at your desired location. Once placed, you will notice a form on the right-hand side of the map, which allows you to configure the Ego car to set its speed, acceleration, along with your desired destination.
+
+![](images/se_ego.png)
+
+The Ego car's heading can also be set by dragging the arrow linked to the ego car
+
+![](images/heading.png)
+
+```
+Note: 
+You can set the ego car’s end point by clicking on the “Add Route Point” icon in the lower right corner of the map. Described in the General Action tools section. Please note that the first version of the Scenario Editor does support the creation of multiple route points for the ego car.
```

Comment:
Remove the last sentence here about multiple route points

Reply:
done